eclair-node/src/main/scala/fr/acinq/eclair/api/FormParamExtractors.scala

```diff
@@ -59,6 +59,8 @@ object FormParamExtractors {
 
   implicit val millisatoshiUnmarshaller: Unmarshaller[String, MilliSatoshi] = Unmarshaller.strict { str => MilliSatoshi(str.toLong) }
 
+  implicit val base64DataUnmarshaller: Unmarshaller[String, ByteVector] = Unmarshaller.strict { str => ByteVector.fromValidBase64(str) }
```

Comment:
How is that being used? It looks like you're not base64-decoding (yet) string inputs from the API

Reply:
Yes, I forgot to remove it.